docs/_steps/creating_workflow_step.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Creating workflow steps
+lang: en
+slug: creating-steps
+order: 2
+---
+
+<div class='section-content'>
+
+To create a workflow step, Bolt provides the `WorkflowStep` class.
+
+When instantiating a new `WorkflowStep`, pass in the step's `callback_id` and a configuration object.
+
+The configuration object contains three keys: `edit`, `save`, and `execute`. Each of these keys must be a single callback or a list of callbacks. All callbacks have access to a `step` object that contains information about the workflow step event.
+
+After instantiating a `WorkflowStep`, you can pass it into `app.step()`. Behind the scenes, your app will listen and respond to the workflow step’s events using the callbacks provided in the configuration object.
+
+</div>
+
+```python
+from slack_bolt import App, WorkflowStep
```

Comment:
@seratch I want to verify that we want to adjust this import to be 

`from slack_bolt.workflows.step import WorkflowStep` 

and not expose it directly off of the package

Reply:
@misscoded If you add only `WorkflowStep` to the top-level, I think it's acceptable as the class is part of the public APIs of the `App` class. I want to mention the same about `AsyncWorkflowSteps` in `slack_bolt.async_app`.

Reply:
We've agreed that adding `WorkflowStep` to the top-level breaks an existing pattern, so for now, we will choose not to do so.